pkg/provider/okta/okta.go

```diff
@@ -1236,3 +1381,214 @@ func fidoWebAuthn(oc *Client, oktaOrgHost string, challengeContext *mfaChallenge
 
 	return gjson.GetBytes(body, "sessionToken").String(), nil
 }
+
+func verifyTrustedCert(oc *Client, doc *goquery.Document, duoHost string, duoSubmitURL string, q url.Values) (*goquery.Document, error) {
```

Comment:
this is the same as the existing PR, just broken out into helper functions to make testing easier